src/app/modules/loop/reducers.ts

```diff
@@ -0,0 +1,211 @@
+import types, { LoopTermsPayload, LoopQuotePayload, LoopPayload } from './types';
+import LoopHttpClient from 'lib/loop-http';
+
+export interface LoopState {
+  loopTerms: null | LoopTermsPayload;
+  loopQuote: null | LoopQuotePayload;
```

Comment:
Are quote and terms ever actually null? Looks like they're initialized with empty fields.

Reply:
Shoots, that's right. Not sure what I was thinking. Removed null